lib/fabrik/DirectorInstance.js

```diff
@@ -73,9 +76,12 @@ class DirectorInstance extends BaseInstance {
       .try(() => {
         switch (operation.type) {
         case 'create':
-          return this
-            .createSecurityGroup(operation.space_guid)
-            .tap(() => operation.state === CONST.OPERATION.SUCCEEDED ? this.scheduleAutoUpdate() : {});
+          if (operation.context.platform === 'cloudfoundry') {
+            return this
+              .createSecurityGroup(operation.context.space_guid)
+              .tap(() => operation.state === CONST.OPERATION.SUCCEEDED ? this.scheduleAutoUpdate() : {});
+          }
+          break;
         case 'update':
           return this.ensureSecurityGroupExists();
```

Comment:
for update the platform specific check seems missing.

Reply:
Missed it. Adding the if before the switch.